### ISSUE TITLE
Some past years not working

### ISSUE THREAD

**Issue by zmoon** [scipy-conference/conference.scipy.org#40]
I don't know if this the right place to report this, but some of the past years are not working on the website.

e.g. <https://conference.scipy.org/scipy2013/> gives a 404 not found page.

I noticed because [this Matplotlib page](https://matplotlib.org/stable/users/resources/index.html) links to one of the talks. Actually clicking on the link there (<https://conference.scipy.org/scipy2013/tutorial_detail.php?id=103>) downloads a PHP file.

**Comment by deniederhut** [comment]
Were you looking for the proceedings from 2013? That should be available at https://conference.scipy.org/proceedings/scipy2013/. If you're looking for the conference page itself, I'm not sure that we were ever hosting it on this server. @dpinte would you know anything about this?

**Comment by zmoon** [comment]
The link was working in Jan 2022 

https://web.archive.org/web/*/https://conference.scipy.org/scipy2013/tutorial_detail.php?id=103

**Comment by dpinte** [comment]
@deniederhut yes, I think I know what is going on. I've transfered the old rackspace server to an S3 bucket serving static content. I was not aware we were still running some php code on that server. I'll see what can be done. Ideally we would migrate it to static content and serve it as html.
